src/newsHeadlines.js

```diff
@@ -0,0 +1,38 @@
+let newsArticles = []
+
+function getNewsTitles() {
+  return fetch("http://news-summary-api.herokuapp.com/guardian?apiRequestUrl=http://content.guardianapis.com/search?order-by=newest&show-fields=body&show-elements=image&page-size=20&show-fields=thumbnail")
+    .then( (response) => {
+      return response.json()
+    })
+}
+
+function extractTitles(dataObject) {
+  let results = dataObject.response.results
+
+  for ( let i = 0; i < results.length; i++ ) {
+    let selected = {
+      webTitle: results[i].webTitle,
+    }
+    newsArticles.push(selected)
+  }
+  return newsArticles
```

Comment:
Saving promise results into an array 🙀 this is GROUNDBREAKING i am so pleased i can study this!!!!

Reply:
I think you could use a forEach here too